osu.Game.Rulesets.Catch/Objects/BananaShower.cs

```diff
@@ -23,29 +23,29 @@ protected override void CreateNestedHitObjects(CancellationToken cancellationTok
 
         private void createBananas(CancellationToken cancellationToken)
         {
-            double spacing = Duration;
+            int startTime = (int)StartTime;
```

Comment:
Probably inline a comment here mentioning the flooring is for stable compatibility.